# Nominal Trajectory
float64[] x
float64[] y
float64[] z
float64[] yaw
float64 time

# For Safety
float64[] x_safe
float64[] y_safe
float64[] z_safe
float64 time_safe
